src/graphql.tsx

```diff
@@ -635,7 +635,10 @@ class ObservableQueryRecycler {
   public recycle (observableQuery: ObservableQuery<any>): void {
     // Stop the query from polling when we recycle. Polling may resume when we
     // reuse it and call `setOptions`.
-    observableQuery.stopPolling();
+    observableQuery.setOptions({
+      fetchPolicy: 'cache-only',
```

Comment:
Setting to `undefined` should be the equivalent of removing. Does this make the implementation easier?

I don’t like the idea of setting `options` manually mainly because we haven’t yet considered all of the implications 😣